FinalProject/Source/Controller/AppDelegate.swift

```diff
@@ -19,9 +19,23 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 	func application(application: UIApplication,
 	                 didFinishLaunchingWithOptions launchOptions: [NSObject: AnyObject]?) -> Bool {
         FIRApp.configure()
+        
+// start test code
+        
+                let myRootRef = FIRDatabase.database().reference()
+
+                // Write data to Firebase
+                myRootRef.setValue("Hi from Nathan :) I just set some data in the backend.")
```

Comment:
@bw-davis check it out! :) We persisted some data to the backend on the root node.